cypress/e2e/ui/settings.cy.js

```diff
@@ -1,6 +1,48 @@
 /* eslint-disable no-undef */
-describe('GTL', () => {
+describe('Settings > My Settings', () => {
   beforeEach(() => {
     cy.login();
+    cy.menu('Settings', 'My Settings');
+  });
+
+  it('Saves the start page setting', () => {
+    cy.get('#display\\.startpage').then((value) => {
+      expect(value[0].value).to.be.oneOf(['', 'Overview / Dashboard']);
+    });
+    cy.get('#display\\.startpage').click();
+    cy.get('.bx--list-box__menu-item__option').then((options) => {
+      const optionArray = Cypress.$.makeArray(options);
+      const match = optionArray.find((el) => el.innerText.trim() === 'Overview / Utilization');
+      if (match) {
+        cy.wrap(match).click();
+      }
+    });
+    cy.intercept('PATCH', '/api/users/1').as('settingsUpdate');
+    cy.get('button').contains('Save').click();
```

Comment:
I like the `cy.contains(selector, string)` generally as cypress will continually try the selector and string query over and over until the timeout is reached.  Whereas `cy.get(selector).contains(string)` will only try the contains(string) over and over until timeout so if the button dom element changes (becomes enabled/disabled/visible/invisible), we might not see that change since we're only doing the `cy.get` once, so we may or may not be able to interact with it.